packages/react/src/components/createInlineOverlayComponent.tsx

```diff
@@ -132,14 +144,8 @@ export const createInlineOverlayComponent = <PropType, ElementType>(
             ? createElement(
                 'div',
                 {
-                  id: 'ion-react-wrapper',
                   ref: this.wrapperRef,
```

Comment:
All overlays in React still need a wrapper because of how we handle teleporting in react